.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: python
+sudo: false
+
+services:
+  - mongodb
+
+addons:
+  apt:
+    sources:
+    - mongodb-3.2-precise
+    packages:
+    - mongodb-org-server
+
+python:
+  - 3.4
+  - 3.5
+
+before_install:
+  - git clone https://github.com/nsls-ii/nsls2-ci ~/scripts
+  - . ~/scripts/install-miniconda.sh ~/
+
+install:
+  - conda create -n testenv --yes nose python=$TRAVIS_PYTHON_VERSION pymongo ujson numpy tornado jsonschema requests pyyaml pytz pytest doct -c lightsource2
+  - source activate testenv
+  - 'pip install https://github.com/NSLS-II/metadatastore/zipball/master#egg=metadatastore'
```

Comment:
I have no idea. I saw it somewhere else in our code base and added it here to get travis to work. Any idea @ericdill?
PS: I need some feature we recently added to mds master for mdservice to work (i.e. no error on import w/o config)